config/plugin/kiam_agent_health.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+OK=0
+NONOK=1
+UNKNOWN=2
+
+CHECK_AGENT="$(curl -s http://169.254.169.254/latest/meta-data/instance-id/)"
```

Comment:
- timeout is missing
- `--fail` curl argument as well as return code check would be desirable to catch non-2XX return codes.

Reply:
Added timeout, leaving out `--fail` so agent response body is not suppressed